### PR TITLE
chore(deps): remove unneeded devDepedencies and switch to c8

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,19 +91,15 @@
     "@netlify/eslint-config-node": "^4.1.2",
     "ava": "^3.0.0",
     "babel-loader": "^8.2.2",
+    "c8": "^7.11.0",
     "cpy": "^8.1.0",
     "cpy-cli": "^3.1.1",
     "cypress": "^7.0.0",
     "del": "^6.0.0",
-    "eslint-plugin-cypress": "^2.11.2",
```
Comment (XhmikosR) [inline package.json:98]: These are included in `@netlify/eslint-config-node`

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 yarn-error.log
-.nyc_output
 coverage
 .eslintcache
 .vscode
 dist
 build
 cypress/videos
 cypress/screenshots
+cypress/support
```
Comment (XhmikosR) [inline .gitignore:10]: Noticed this new directory added after running the tests

File: package.json (hunk 1 above)
Comment (ehmicky) [reply package.json:98]: Good catch!

**Review by ehmicky — APPROVED** [review]
Thanks a lot @XhmikosR, this is good timing as I'm planning on working on that repository today :)

**Pull request merged**